src/PythonMigrationViewExtension/MigrationAssistant/custom_fixers/fix_none.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2006 Google, Inc. All Rights Reserved.
```

Comment:
is this really copyright google?

Reply:
I copy-pasted from another fixer. Will remove.